scripts/build/download-tool.sh

```diff
@@ -78,7 +78,7 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+wget --no-verbose --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT" 2>&1
```

Comment:
Oh, that's probably because MinGW doesn't ship root certs and doesn't point to existing ones either. Maybe `curl` works (I doubt it)? If nothing does, we could add a `.js` script that uses Node's `http(s)` to download stuff, as that'll have everything available & will work on any platform.